src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -6,19 +6,104 @@
  * За бажанням можна реалізувати інші методи інтрефейсу Map.</p>
  */
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    static final int DEFAULT_CAPACITY = 16;
+    static final float DEFAULT_FACTOR = 0.75f;
+    private Node<K, V>[] table;
+    private int size = 0;
+
+    public MyHashMap() {
+        table = (Node<K, V>[]) new Node[DEFAULT_CAPACITY];
```

Comment:
```suggestion
        table = new Node[DEFAULT_CAPACITY];
```
повинно працювати так

Reply:
Done